andela/gradr
# #166974743 Clone a SPEC

#### What does this PR do?
- adds clone icon to the HTML file
- adds a method to clone a spec

#### Description of Task to be completed
- Admins should be able to re-use SPECs when they want to create one that closely mimics an existing SPEC

#### How should this be manually tested?
- switch to the branch `ft-clone-spec-166974743`
- start the server using `yarn develop-admin`
- open an existing spec and clone it.
- Notice the name of the spec changes and the spec is added to the list of existing SPECs

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#166974743 Clone a SPEC](https://www.pivotaltracker.com/story/show/166974743)

#### Screenshots (if appropriate)
> Original SPEC
![image](https://user-images.githubusercontent.com/28973383/60585570-4fbabb80-9d90-11e9-9aaf-1ccdf156b37f.png)

> Cloned SPEC
![image](https://user-images.githubusercontent.com/28973383/60585678-8bee1c00-9d90-11e9-8f9d-3ab43031e094.png)
